code/modules/projectiles/projectile/special/goat.dm

```diff
@@ -0,0 +1,18 @@
+/obj/projectile/goat
+	name = "goat"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "goat"
+	damage = 20
+	damage_type = BRUTE
+	flag = "bullet"
+
+/obj/projectile/goat/on_hit(atom/target)
+	if(target == firer)
+		forceMove(target.loc)
+		return
+	paralyze = 20
+	knockdown = 40
+	var/turf/location = get_turf(target)
+	new/mob/living/simple_animal/hostile/retaliate/goat(location)
```

Comment:
these goats won't immediately begin attacking your target; they're just gonna mill around for a bit until they succeed on their "start attacking people" roll.

Reply:
goats do whateves goats do

Reply:
ay its goats man they do what they want